### PR TITLE
Refactor admin/seeder to use Seeder Service

### PR DESCRIPTION
**Pull request by zoltan-nz**
- Removed Fader Label which used an Observer.
- Most of the logic moved in a service.

### CODE REVIEW

File: database.rules.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
   "rules": {
     ".read": true,
-    ".write": true
+    ".write": true,
+    "books": {
+      ".indexOn": ["library","author"]
+    }
```
Comment (tkraak) [inline database.rules.json]: What does this do?

File: tests/unit/services/seeder-test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Service | seeder', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
+    let service = this.owner.lookup('service:seeder');
+    assert.ok(service);
```
Comment (tkraak) [inline tests/unit/services/seeder-test.js]: I can fix up our existing tests for the services if you want to merge this first.

File: database.rules.json (hunk 1 above)
Comment (zoltan-nz) [reply database.rules.json]: Firebase generated a few warnings in console and suggested to add `.indexOn` to Firebase config. Probably the database engine adds indexes to documents so the lookup for referenced record can be faster.

Comment (zoltan-nz) [reply database.rules.json]: More information about `.indexOn` on Firebase: https://firebase.google.com/docs/database/security/indexing-data

File: tests/unit/services/seeder-test.js (hunk 2 above)
Comment (zoltan-nz) [reply tests/unit/services/seeder-test.js]: Yep, let's merge this and add tests to service in a separated PR.

**Pull request merged**